index.js

```diff
@@ -16,7 +15,8 @@ module.exports = {
   // rangeEnd: Infinity,
   // requirePragma: false,
   // insertPragma: false,
-  // proseWrap: "preserve",
+  proseWrap: "always",
+  printWidth: 80,
```

Comment:
[80 is default print width](https://prettier.io/docs/en/options.html#print-width), so this won't change anything. Let's keep defaults commented out as before if you didn't have any other specific reason of adding it 🙂

Reply:
done!